### DISCUSSION TITLE
yyg/ca46b09/

### DISCUSSION THREAD

**Discussion by giscus** [MarsperL/MarsperL.github.io#21] [Announcements]
# yyg/ca46b09/

前言平时在工作和生活中，有一些很好用的软件、网站，就想着找一个地方都给记下来。在创建hexo博客时，看到了有在博客里添加网站导航的，感觉这样挺不错的，也就有了繁沙：一个搜录（个人）好用软件和网站的地方。样式借用了Leonus 博主的作品推荐卡片—标签外挂 | Leonus，做了一点修改适配：  卡片盒子可一行放多个  盒内的提示文本内容转成了一个外部独立显示的提示框文本（尚待优化，若有好的优化方式

http://localhost:4000/yyg/ca46b09/

**Comment by MarsperL** [comment]
😂
